.env.example

```diff
@@ -0,0 +1,4 @@
+PORT=4000
+DATABASE_URL="postgresql://boolean_owner:TZ0rnzWdmbu8@ep-morning-cell-a24xw8zh-pooler.eu-central-1.aws.neon.tech/team-dev-sim?sslmode=require"
+JWT_SECRET="somesecurestring"
+JWT_EXPIRY="24h"
```

Comment:
We can't merge this while there's a real database connection string in a config file like this. Additionally, you'll need to completely remove the commit that added this from the commit history otherwise it will still exist in a commit on the main branch